app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -33,6 +35,13 @@ export const CollectionPage = (): React.ReactElement => {
     updateCollection();
   }, [updateCollection]);
 
+  React.useEffect((): void => {
+    setCollectionStatistics(null);
```

Comment:
this is wrong - it should be undefined whilst loading and then null only if there is an error